doc/api/addons.markdown

```diff
@@ -868,6 +939,19 @@ Test in JavaScript by running:
     // test.js
     const addon = require('./build/Release/addon');
 
+## Native Abstractions for Node.js
+
+Each of the examples illustrated here make direct use of the Node.js and V8
+APIs for implementing Addons. It is important to note that Addons written
+directly to the V8 APIs can be directly impacted by periodic changes made to
+the V8 library.
+
+The [Native Abstrations for Node.js](https://github.com/nodejs/nan)
+(or `nan`) provide a set of tools that Addon developers can use to more easily
+adapt to changes in the V8 APIs. See the `nan`
+[examples](https://github.com/nodejs/nan/tree/master/examples/async_pi_estimate)
```

Comment:
move to footer link to be consistent, also best to link to just the /examples/ directory instead of that specific example, there's discussion that we may want to remove that one in favour of some kind of simple map/reduce that better demonstrates what's going on